nwbinspector/checks/general.py

```diff
@@ -0,0 +1,23 @@
+"""Check functions that examine any general neurodata_type with the available attrbutes."""
+from ..register_checks import register_check, InspectorMessage, Importance
+
+COMMON_DESCRIPTION_PLACEHOLDERS = ["no description", "no desc", "none"]
+
+
+@register_check(importance=Importance.CRITICAL, neurodata_type=None)
+def check_name_slashes(obj):
+    """Check if there  has been added for the session."""
+    if hasattr(obj, "name") and any((x in obj.name for x in ["/", "\\", "|"])):
```

Comment:
Do we care about the backslashes and the pipe? What is the reasoning behind disallowing those? We need to be very careful about CRITICAL checks

Reply:
The pipe wouldn't matter, TBH. I can remove that.

The backslash can create different behavior depending on how the string name is parsed by downstream code, *e.g.*, `name1="ElectricalSeries\First" is fine, but `name2="ElectricalSeries\number2" would parse it as a newline separation.

Reply:
ah. hmm. I'll think about that.